src/tests/dates.test.ts

```diff
@@ -21,6 +28,79 @@ describe('abbreviationForWeekday()', () => {
   });
 });
 
+describe('getDateDiff()', () => {
+  let now;
+  let testDate;
+
+  beforeEach(() => {
+    now = new Date();
+    testDate = new Date(now.getTime());
```

Comment:
Not sure about this one, but I prefer the pattern you have in your other tests where you declare this within the tests itself.